examples/np/cma_es_cartpole.py

```diff
@@ -16,6 +18,20 @@
 from garage.tf.experiment import LocalTFRunner
 from garage.tf.policies import CategoricalMLPPolicy
 
+parser = argparse.ArgumentParser()
+parser.add_argument('--n_epochs',
+                    type=int,
+                    default=100,
+                    help='Number of epochs to run')
+args = parser.parse_args()
+
+parser = argparse.ArgumentParser()
+parser.add_argument('--n_epochs',
```

Comment:
this is repeated. is that intentional?

Reply:
Ah. Good catch. Not intentional. Somehow got added twice during a regex replace op